cmd/ferretdb/main.go

```diff
@@ -83,7 +82,7 @@ func main() {
 		logger.Sugar().Warn("The current TLS implementation is not secure.")
 	}
 
-	ctx, stop := signal.NotifyContext(context.Background(), unix.SIGTERM, unix.SIGINT)
```

Comment:
why have you changed from unix to syscall?

Reply:
Didn't know that syscall is deprecated.